ClusteringModel.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+
+
+class ClusteringModel(ABC):
+    @abstractmethod
+    def getClusters(self, documents):
```

Comment:
Consider adding type hints (especially for interface definitions) as it makes it much easier to add new derived classes and also ensure that existing derived classes are correct.

Reply:
makes sense - will add